README.md

```diff
@@ -402,11 +402,17 @@ See this repo's [header.yml](_data/header.yml) for more info.
 
 ### Footer
 
-The [footer.html include](_includes/footer.html) sets the `header` template variable to `site.data.footer`, the value of which is set in your Jekyll project's `_data/footer.yml` file. Then it includes [components/footer.html](_includes/components/footer.html) to
+The [footer.html include](_includes/footer.html) sets the `footer` template variable to `site.data.footer`, the value of which is set in your Jekyll project's `_data/footer.yml` file. Then it includes [components/footer.html](_includes/components/footer.html) to
 render the footer's markup.
 
 See this repo's [footer.yml](_data/footer.yml) for more info.
 
+### Identifier
```

Comment:
Can we split the addition of the Identifier out to a separate pull request? Can save the discussion for there, but in short, I worry it's over-complicating the theme — adding an alternative to the Footer instead of replacing it — when [we plan to go the other direction](https://github.com/18F/uswds-jekyll/issues/190).

Reply:
Sure, we can discuss it there

Reply:
Still need to remove this part, I think 😉